src/explore-education-statistics-admin/src/pages/release/content/components/BasicReleaseSummary.tsx

```diff
@@ -60,11 +60,7 @@ const BasicReleaseSummary = ({ release }: Props) => {
               <Tag className="govuk-!-margin-right-3 govuk-!-margin-bottom-3">
                 {getReleaseApprovalStatusLabel(release.approvalStatus)}
               </Tag>
-              {release.type.title !== 'Transparency Data' ? (
-                <Tag>{release.type.title} statistics</Tag>
-              ) : (
-                <Tag>{release.type.title}</Tag>
-              )}
+              <Tag>{release.type.title}</Tag>
```

Comment:
not sure if we want Management Information to have statistics appended to it but if we don't we will need to add a ternary op to check for this
```
{release.type.title !== 'Management Information' ? <Tag>{release.type.title} statistics</Tag> : <Tag>{release.type.title}</Tag>}
```